client/my-sites/checkout/index.js

```diff
@@ -56,7 +55,7 @@ export default function () {
 	page(
 		'/checkout/jetpack/thank-you/licensing-manual-activate/:product',
 		noSite,
-		licensingThankYouManualActivation,
```

Comment:
Just one question here, if this gets deployed. the checkout flow in the production will be affected immediately.

Do we need to use `jetpack/standalone-plugin-onboarding-update-v1` feature flag here?

Reply:
`LicensingActivationInstructions` already uses the feature flag, so I think it's good to go.